### ISSUE TITLE
Bug: If you drag an item from one category to another, it doesn't stick

### ISSUE THREAD

**Issue by klreeher** [galenmaly/lighterpack#71]
Found this on https://lighterpack.com so if that's behind and this is fixed already, my bad. Win 10, latest Chrome.

If you drag an item from one category to another, but don't drag the item above the "add new item" button/line, the item will stick in the new category but will appear below the "add new item" line. On page refresh, the item will appear back in the original category.

**Comment by johnabela** [comment]
It would be really helpful if you provided what browser, and version of said browser, you are using.

I have attempted to reduplicate this, _[Chrome | Version 59.0.3071.115 (Official Build) (64-bit)]_

When I copy a piece of gear from one category to another category, placing it **below** the _Add new item_ UI button/link inside of the move-to category, it (a) removes the item from the original category, and (b) places it below the _Add new item_ UI link/button.

If I then refresh the page, the item is (a) within the moved-to category, **above** the _Add new item_ link/button, and (b) has been removed from the moved-from _(original)_ category.

So for me, the differences between my testing and that of @klreeher is that of action **(b)** on page reload.
